Guide-GroundSystem.txt

```diff
@@ -23,59 +23,17 @@ Before launching the Ground System make sure that:
   -> cmdUtil is compiled.
 
 
-Installing and running cFS Ground System on Mac:
-Install using Homebrew on Mac (http://brew.sh/)
-    $ brew install pyqt
-    $ brew install zeromq
-    $ ( cd Subsystems/cmdUtil/ && make )
-    $ python GroundSystem.py
-
 Installing and running cFS Ground System on Ubuntu:
-    $ sudo apt-get install python-qt4 python-qt4-dev
-    $ sudo apt-get install python-zmq
+    $ sudo apt-get install python3-pyqt4
+    $ sudo apt-get install python3-zmq
+    $ sudo apt-get install libcanberra-gtk-module
     $ ( cd Subsystems/cmdUtil/ && make )
-    $ python GroundSystem.py
+    $ python3 GroundSystem.py
 
 
-Installing and running cFS Ground System on CentOS 6:
```

Comment:
Add these back in and note that they're outdated.

Reply:
I made the changes yesterday.  We should be good to go.